src/Packages/Passport/Runtime/Scripts/Private/PassportImpl.cs

```diff
@@ -703,6 +703,16 @@ public async UniTask<string> ZkEvmSendTransaction(TransactionRequest request)
             return callResponse.GetStringResult();
         }
 
+        public async UniTask<TransactionReceiptResponse?> ZkEvmGetTransactionReceipt(string hash)
```

Comment:
Change `TransactionReceiptResponse?` to `TransactionReceiptResponse` as this does not work for Unity 2019.